src/App.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import { Card } from './components/Card';
 import './style.css';
+import logo from './images/wifi.png';
 
 function App() {
   return (
     <div className="App">
       <h1>
-        <span role="img" aria-label="antenna-bars">
-          📶
-        </span>
+      <img src={logo} width="32" height="32" />
```

Comment:
The local react development server is complaining about a missing `alt` prop, mind adding one?

```
src/App.js
  Line 10:9:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
```